src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,30 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import java.io.IOException; // OK
+ * import java.net.URL; // OK
+ *
+ * import javax.net.ssl.SSLContext; // OK
+ * import javax.net.ssl.TrustManager; // OK
+ *
+ * import org.apache.http.conn.ClientConnectionManager; // OK
+ * import org.apache.http.conn.scheme.Scheme; // OK
+ *
+ * import com.neurologic.http.HttpClient; // OK
+ * import com.neurologic.http.impl.ApacheHttpClient; // OK
+ *
```

Comment:
Please show some violations in this example.

Reply:
done

Reply:
Resolved